tests/fixtures.py

```diff
@@ -118,6 +118,7 @@ def tempfile_path():
     """
     User is responsible for closing the file given at the path.
     """
-    _, path = tempfile.mkstemp()
+    fd, path = tempfile.mkstemp()
     yield path
+    os.close(fd)
```

Comment:
So, Windows doesn't complain that this file is opened elsewhere?

Reply:
File opened in the same process using `with` statement, I'm not sure why it still open